etcdserver/api/v3rpc/maintenance.go

```diff
@@ -128,11 +128,11 @@ func (ms *maintenanceServer) Snapshot(sr *pb.SnapshotRequest, srv pb.Maintenance
 }
 
 func (ms *maintenanceServer) Hash(ctx context.Context, r *pb.HashRequest) (*pb.HashResponse, error) {
```

Comment:
probably a different request type/resp makes more sense since Maintenance Hash API shouldn't have reason about `rev` in `HashRequest`  nor `compact_revision` in `HashResponse`